apps/web-giddh/src/assets/css/style-2.scss

```diff
@@ -9978,6 +9978,15 @@ input.mat-input-element{
     }
 
   @media (max-width: 768px) {
+        .menu_open .sales-register{
```

Comment:
Please check if mentioning only .sales-register would work as the styles are redundant